requirements/edx/base.txt

```diff
@@ -79,14 +79,14 @@ python-social-auth==0.2.12
 pytz==2015.2
 pysrt==0.4.7
 PyYAML==3.10
-requests==2.7.0
+requests==2.9.1
```

Comment:
@feanil sorry I'm a new contributor, can you clarify what you need here? CHANGELOG.rst just points to the release notes and git commit history. Where should I document this change?

This update is a requirement of [python-keystoneclient](https://github.com/openstack/python-keystoneclient/blob/master/requirements.txt), which is itself a dependency of [django-storage-swift](https://github.com/blacktorn/django-storage-swift/blob/master/setup.py#L15).